.storybook/middleware.js

```diff
@@ -9,16 +9,19 @@ const extensionRegex = new RegExp(`${fixedExtension}`);
 
 module.exports = function expressMiddleware(app) {
   app.use(function (req, res, next) {
-    if (/mock\/[a-zA-Z0-9-_]*.json/.test(req.url)) {
-      res.setHeader('Content-Type', 'application/json');
+    if (/mock\/[a-zA-Z0-9-_]*.(json|xml)/.test(req.url)) {
+      const isXml = /\.xml/.test(req.url);
+      const contentType = isXml ? 'application/xml' : 'application/json';
+
+      res.setHeader('Content-Type', contentType);
       console.log('[POST => GET] : ' + req.url);
```

Comment:
Wird dieser Log benötigt?

Reply:
Ja der teilt dir in der Konsole mit, dass dein Fake POST in einen GET umgewandelt wurde und welche request url du aufrufen wolltest. Finde ich recht hilfreich.